### ISSUE TITLE
Test for Iconbutton

### ISSUE THREAD

**Issue by sazan10** [prokuranepal/inventory_app#42]
<b> IconButton needs test </b>
<a href="https://github.com/prokuranepal/inventory_app/blob/inventory/src/components/Component/IconButton.js"> IconButton.js</a>

<b> The tests should perform at least</b>
<ul>
<li>Component testing for the components present and their numbers </li>
<li>Simulation for events like Button Press</li>
<li> Props testing </li>
</ul>
The project tests are based on jest and enzyme. Tests like <a href="https://github.com/prokuranepal/inventory_app/blob/inventory/src/_test_/componenets/ManageItem.test.js">test1 </a> or <a href="https://github.com/prokuranepal/inventory_app/blob/inventory/src/_test_/componenets/ManageItem.test.js"> test2 </a> could serve as references.

**Comment by Coutinho100** [comment]
I would like to work on this @sazan10.

**Comment by sazan10** [comment]
Sure @Coutinho100, I have assigned the issue to you.

**Comment by Coutinho100** [comment]
Seems like you have used <b>Platform</b> in the IconButton. Should I mock it or have you been handling it in a different way?

**Comment by sazan10** [comment]
Yes, we have mocked it as well. You could refer <a href="https://github.com/prokuranepal/inventory_app/blob/inventory/src/_test_/componenets/test_function.js"> test_function</a> for platform mocks as well as other mocks and functions🙂
